ert_shared/models/base_run_model.py

```diff
@@ -319,6 +326,136 @@ def run_ensemble_evaluator(
         run_context.get_sim_fs().fsync()
         return totalOk
 
+    async def _evaluate(
+        self, run_context: ErtRunContext, ee_config: EvaluatorServerConfig
+    ) -> None:
+        """Start asynchronous evaluation of an ensemble."""
+        experiment_logger.debug("_evaluate")
+        loop = asyncio.get_running_loop()
+        if run_context.get_step():
+            self.ert().eclConfig().assert_restart()
+        experiment_logger.debug("building...")
+        ensemble = EnsembleBuilder.from_legacy(
+            run_context,
+            self.get_forward_model(),
+            self._queue_config,
+            self.ert().analysisConfig(),
+            self.ert().resConfig(),
+        ).build()
+        experiment_logger.debug("built")
+
+        ensemble_listener = asyncio.create_task(
```

Comment:
Just wanted to write why not to use `asyncio.ensure_future` but then I saw it calls `asyncio.create_task` internally :)

Reply:
Right, the docs also recommend `create_task` over `ensure_future`.